OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for iqserver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```

Comment:
As I questioned the need for replicaCount previously, I *think* I saw that this value may not actually be used in the templates.